tasks/main.yml

```diff
@@ -16,15 +20,21 @@
 
 - name: Download Caddy
   get_url: url=https://caddyserver.com/download/build?os=linux&arch=amd64&features={{ caddy_features }} dest={{ caddy_home }}/caddy.tar.gz force=yes
-  when: caddy_releases_cache.changed or caddy_features_cache.changed
-  register: caddy_binary_cache
+  when: caddy_update and (caddy_releases_cache.changed or caddy_features_cache.changed)
```

Comment:
I understand this line of 42364a9faf8152260413d3d2a6aea49fdd9bcc20 is what the commit message part "Also clarify conditions for download." of 42364a9faf8152260413d3d2a6aea49fdd9bcc20 refers to.

I understand that when preparing c45379b627dbad24e1c26efa06a91f43a5434842 I felt that explicating implicit dependencies among variables would have eased readability hence maintenance.